numpy/lib/type_check.py

```diff
@@ -9,8 +9,8 @@
            'common_type']
 
 import numpy.core.numeric as _nx
-from numpy.core.numeric import asarray, asanyarray, array, isnan, \
-                obj2sctype, zeros
+from numpy.core.numeric import (asarray, asanyarray, array, isnan,
+                                obj2sctype, zeros, isscalar)
```

Comment:
Given that we imported `_nx` on the line above, and much of this file uses it already, is there any real point importing `isscalar` here?